views/default.jade

```diff
@@ -0,0 +1,20 @@
+doctype html
+html(lang='en', ng-app='app' ng-controller='AppController')
+  head
+    meta(charset='utf-8')
+    meta(http-equiv='X-UA-Compatible', content='IE=edge')
+    meta(http-equiv='Content-Language', content='en')
+    meta(name='viewport', content='width=1020')
+    meta(name='description', content='something about a game')
+    link(rel='stylesheet', href='https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,light,bolditalic,black,medium&lang=en')
+    link(rel='stylesheet', href='https://fonts.googleapis.com/icon?family=Material+Icons')
+    link(rel='stylesheet', href='/style/site.css')
+    script(src='/js/vendor.min.js')
+    block title
+      title War Room - Cards
+    block head
```

Comment:
`block head` has no default contents - it can be injected later when `extends` is used ... see `views/index.jade`